alternator/executor.cc

```diff
@@ -1206,10 +1249,10 @@ future<executor::request_return_type> executor::update_table(client_state& clien
     elogger.trace("Updating table {}", request);
 
     static const std::vector<sstring> unsupported = {
-        "GlobalSecondaryIndexUpdates", 
+        "GlobalSecondaryIndexUpdates",
```

Comment:
Again, please avoid unrelated whitespace changes (or leave them to a different patch).